conf/modules.config

```diff
@@ -409,7 +456,7 @@ process {
         ]
     }
 
-    withName: GTDBTK_CLASSIFY {
+    withName: GTDBTK_CLASSIFYWF {
```

Comment:
The name addition `WF` seems here out of place. Is that correct?

Reply:
That's called what it is in the module I think?

Reply:
Indeed, weird naming for a module.